LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/no-painting-while-render-blocked.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html class=reftest-wait>
+<title>View transitions: Rendering suppression prevents painting</title>
+<link rel="help" href="https://github.com/WICG/view-transitions">
```

Comment:
https://drafts.csswg.org/css-view-transitions-1/#document-rendering-suppression-for-view-transitions would probably be a better link.